### PR TITLE
Added constraints 2,6,13 Changed checkAllInstancesConstraints function

### PR DESCRIPTION
**Pull request by JonatanMilver**
...in every given class

### CODE REVIEW

File: Booking.java [hunk 1]
```diff
@@ -1,6 +1,5 @@
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
+import java.net.ServerSocket;
+import java.util.*;
```
Comment (JonatanMilver) [inline Booking.java:2]: these 2 line were probably added by mistake.

File: Booking.java [hunk 2]
```diff
@@ -56,10 +55,33 @@ public Review getReview() {
...
+        for(HotelService service: this.services) {
+            boolean found = false;
+            for (HotelService hotelService : hotelServices.values()) {
+                if (service == hotelService) found = true;
+            }
+            if (!found) return false;
+        }
+        return true;
+
+    }
```
Comment (tomerkel) [inline Booking.java:86]: you sure that this is constraint 13?
you need to compare the hotels, no?

**Review by tomerkel — APPROVED** [review]

File: Booking.java (hunk 2 above)
Comment (JonatanMilver) [reply Booking.java:86]: It's constraint number 13 in part B.
Anyways, the constraint checks the services in a specific booking(which is booked to a specific hotel) are given by this specific hotel. In line 76 it get's a specific hotel's services - which means there is a comparison thus there shouldn't be another comparison.

File: Booking.java [hunk 3]
```diff
@@ -56,10 +55,33 @@ public Review getReview() {
 
     @Override
     public boolean checkConstraints() {
-        return true;
+        if(constraint_13()) return true;
```
Comment (Guyzaidman) [inline Booking.java:58]: if method constraint_13 is boolean you can simply do:
return constraint_13()
but its readabilty only

**Review by Guyzaidman — APPROVED** [review]

File: Booking.java (hunk 3 above)
Comment (JonatanMilver) [reply Booking.java:58]: I didn't do it because there might be more constraints added to this function by you guys so all should be with IF statements

**Pull request merged**